### DISCUSSION TITLE
Access HTML context variables

### DISCUSSION THREAD

**Discussion by j2bbayle** [executablebooks/meta#969] [Q&A]
First of all, thanks so very much for this super-useful package!

I'm currently building a multi-version documentation, mostly written in Markdown using [myst-parser](https://myst-parser.readthedocs.io/en/latest/index.html). Everything works fine, except that I can't seem to be able to use substitutions (https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#substitutions-with-jinja2) to access the current_version variable.

I've made sure that 'substitution' was in my myst_enable_extensions, and I've tried to use

```
{{ current_version.name }}
```

as well as

```
{{ env.config.current_version.name }}
```

but none of these are defined.

Is there a way to access this variable (defined in the HTML context)?

Thanks a lot!

**Comment by chrisjsewell** [comment]
Heya cheers,

As it shows in the docs, I know you can do:

```
{{ env.config.version }}
```

Is this what you are after, or is `current_version` something else?

**Reply by j2bbayle** [reply]
I think it is something else, as `current_version` will change to the version multi-version is currently trying to build (one build run produces multiple versions of the documentation, e.g. one for each branch) — while `env.config.version` keeps its value throughout all versions. 

Not sure I’m clear here though 😅.
